scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -326,9 +347,46 @@ public void exportJobXML(String jobId) {
                     LogModel.getInstance().logMessage("Downloading XML for job: " + jobId);
                     Window.open(jobXmlUrl, "Download Job XML", "");
                 } else {
-                    LogModel.getInstance()
-                            .logMessage("Could not download Job XML. Got HTTP response code: " + xhr.getStatus());
-                    SC.warn("Could not export job's XML:\n Got HTTP response with error code " + xhr.getStatus());
+                    // Check if it's a user-permission related error.
+                    String error = xhr.getResponseHeader(HEADER_PA_ERROR);
+                    if (xhr.getResponseHeader(HEADER_PA_ERROR) != null) {
```

Comment:
```suggestion
                    if (error != null) {
```

Reply:
yes that's why I created error in the first place ! I forgot that :(((